test/libsolidity/SyntaxTest.cpp

```diff
@@ -56,41 +57,78 @@ SyntaxTest::SyntaxTest(string const& _filename)
 	m_expectations = parseExpectations(file);
 }
 
-bool SyntaxTest::run(ostream& _stream, string const& _indent)
+bool SyntaxTest::run(ostream& _stream, string const& _linePrefix, int _indent, bool const _formatted)
 {
 	m_errorList = parseAnalyseAndReturnError(m_source, true, true, true).second;
 	if (!matchesExpectations(m_errorList))
 	{
-		std::string nextIndentLevel = _indent + "\t";
-		_stream << _indent << "Expected result:" << endl;
-		printExpected(_stream, nextIndentLevel);
-		_stream << _indent << "Obtained result:\n";
-		printErrorList(_stream, m_errorList, nextIndentLevel);
+		std::string nextIndentLevel = _linePrefix + (_indent < 0 ? string(-_indent, '\t') : string(_indent, ' '));
```

Comment:
What is the purpose of negative indent?

Reply:
The idea is to use the SyntaxTest class both in the boost tests themselves and in the syntax test tool ``isoltest``, so that they can't differ in behaviour.

For the boost test it seemed more appropriate to use tabs for indentation, though, whereas the output of isoltest is much more readable with only two spaces as indentation - that's why SyntaxTest supports both.

Reply:
Are you sure this is worth the hassle? Why not use spaces everywhere?

Reply:
Well, if you're fine with two spaces in the boost tests, then I can use them everywhere - although the hassle is basically only this one line.